### ISSUE TITLE
Android - UI - Detail Screen: Only show delete icon when adding or editing item

### ISSUE THREAD

**Issue by ceceliacreates** [WomenWhoCode/WWCodeMobile#94]
- Hide delete button when adding a new entry
- Display delete button when viewing details/editing an existing entry

<img width="273" alt="Screenshot 2024-01-24 at 5 30 44 PM" src="https://github.com/WomenWhoCode/WWCodeMobile/assets/40367173/fe7de29e-7d89-47a5-b91f-ca00a903bed4">

**Comment by jyoshna-joshi** [comment]
Hi, I would like to work on this issue once unblocked. Thanks

**Comment by jyoshna-joshi** [comment]
@renweview @gaurigadkari

**Comment by jyoshna-joshi** [comment]
Hi @ceceliacreates, I am working on another issue right now. If anyone is interested in this issue, please reassign. Otherwise, I can do it, once I am finished with current issue.

**Comment by pelumy** [comment]
@ceceliacreates and @devanshimodha , I will like to pick this up.

**Comment by ceceliacreates** [comment]
Assigned you @pelumy !

**Comment by pelumy** [comment]
> Assigned you @pelumy !

@ceceliacreates, I am so sorry. I thought this was for the iOS project, I just realized it is for the android. I don’t have enough space on my laptop to also install android studio.

I will like to be unassigned 😢

**Comment by ceceliacreates** [comment]
No problem!! I unassigned.

**Comment by pelumy** [comment]
Thank you @ceceliacreates

**Comment by jyoshna-joshi** [comment]
@ceceliacreates I would like to work on this one. Can you please assign this to me?

**Comment by ThanaReka** [comment]
Hi @baghaii, I would like to work on this issue next, once my current PR has been merged. Thanks!

**Comment by baghaii** [comment]
@jyoshna-joshi , I have assigned this to you. 

Can you include the ability to click on rows to get to the details screen as part of this work? That work was not completed in earlier tickets.

**Comment by jyoshna-joshi** [comment]
Sure. Will work on that too.
